### PR TITLE
add slayer from guilty gear xrd rev 2

### PR DESCRIPTION
**Pull request by yithian**

### CODE REVIEW

File: src/data/guilty_gear_xrd_rev_2/slayer.yml [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+moves:
+  normals:
+    - 5P
+    - 5K
+    - c.S
+    - f.S
+    - 5H
+    - 6P
+    - 6K
+    - '6[K]'
```
Comment (komidore64) [inline src/data/guilty_gear_xrd_rev_2/slayer.yml]: Where is it shown that `6[K]` is a valid normal?

Comment (yithian) [reply src/data/guilty_gear_xrd_rev_2/slayer.yml]: I'm kind of irritated by this, actually. It's in the text for his 6K, not as a separate move.

`Hold down K to feint, 6[K]`

Comment (komidore64) [reply src/data/guilty_gear_xrd_rev_2/slayer.yml]: gross

Comment (komidore64) [reply src/data/guilty_gear_xrd_rev_2/slayer.yml]: i see it now.

**Comment by komidore64** [comment]
looks good my dude.

ACK

**Pull request merged**